src/write_redis.c

```diff
@@ -67,7 +68,12 @@ static int wr_write (const data_set_t *ds, /* {{{ */
   status = FORMAT_VL (ident, sizeof (ident), vl);
   if (status != 0)
     return (status);
-  ssnprintf (key, sizeof (key), "collectd/%s", ident);
+  if (node->prefix == NULL) {
+    ssnprintf (key, sizeof (key), "collectd/%s", ident);
+  }
+  else {
+    ssnprintf (key, sizeof (key), "%s/collectd/%s", node->prefix, ident);
```

Comment:
Other plugins, for example the *write_graphite plugin*, do **not** add a separator after the prefix, so users have more flexibility. I'd prefer if this would follow the same style.